types/hibp/index.d.ts

```diff
@@ -0,0 +1,97 @@
+// Type definitions for hibp 7.2
+// Project: https://github.com/wkovacs64/hibp
+// Definitions by: Silas Rech <https://github.com/lenovouser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
+
+export interface BreachOptions {
+    domain?: string;
+    truncate?: boolean;
+}
+
+export interface Breach {
```

Comment:
None of these properties are optional?

Reply:
No, the database is manually maintained by @troyhunt and the responses always contain all of these properties as far as I know.

[haveibeenpwned.com/api/v2#BreachModel](https://haveibeenpwned.com/API/v2#BreachModel)